pkg/chartutil/requirements.go

```diff
@@ -266,7 +267,7 @@ func ProcessRequirementsEnabled(c *chart.Chart, v *chart.Config) error {
 	for _, existingDependency := range c.Dependencies {
 		var dependencyFound bool
 		for _, req := range reqs.Dependencies {
-			if existingDependency.Metadata.Name == req.Name && existingDependency.Metadata.Version == req.Version {
+			if existingDependency.Metadata.Name == req.Name && version.IsCompatibleRange(req.Version, existingDependency.Metadata.Version) {
```

Comment:
same here. Would you mind writing up a unit test to cover this?

Reply:
I have added the unit test,please review it@bacongobbler

Reply:
I have added the unit test,please review it